app/Filament/Resources/GameResource/Pages/SimilarGames.php

```diff
@@ -0,0 +1,206 @@
+                        false: fn (Builder $query): Builder => $query->where('achievements_published', '<', 6),
+                        blank: fn (Builder $query): Builder => $query,
+                    ),
+            ])
+            ->headerActions([
+                Tables\Actions\Action::make('add')
+                    ->label('Add similar games')
+                    ->form([
+                        Forms\Components\Select::make('game_ids')
+                            ->label('Games')
+                            ->multiple()
+                            ->options(function () {
+                                return Game::whereNot('ID', $this->getOwnerRecord()->id)
+                                    ->limit(50)
+                                    ->with('system')
+                                    ->get()
+                                    ->mapWithKeys(fn ($game) => [$game->id => "[{$game->id}] {$game->title} ({$game->system->name})"]);
+                            })
+                            ->getOptionLabelsUsing(function (array $values): array {
+                                return Game::whereIn('ID', $values)
+                                    ->with('system')
+                                    ->get()
+                                    ->mapWithKeys(fn ($game) => [$game->id => "[{$game->id}] {$game->title} ({$game->system->name})"])
+                                    ->toArray();
+                            })
+                            ->searchable()
+                            ->getSearchResultsUsing(function (string $search) {
+                                return Game::whereNot('ID', $this->getOwnerRecord()->id)
+                                    ->where(function ($query) use ($search) {
+                                        $query->where('ID', 'LIKE', "%{$search}%")
+                                            ->orWhere('Title', 'LIKE', "%{$search}%");
+                                    })
+                                    ->limit(50)
+                                    ->with('system')
+                                    ->get()
+                                    ->mapWithKeys(fn ($game) => [$game->id => "[{$game->id}] {$game->title} ({$game->system->name})"]);
+                            }),
+                    ])
+                    ->modalHeading('Add similar games')
+                    ->modalAutofocus(false)
+                    ->action(function (array $data): void {
+                        /** @var Game $game */
+                        $game = $this->getOwnerRecord();
+
+                        // This assumes there will only ever be one similar games set.
+                        $similarGamesSet = GameSet::whereType(GameSetType::SimilarGames)
+                            ->whereGameId($game->id)
+                            ->first();
+
+                        (new AttachGamesToGameSetAction())->execute($similarGamesSet, $data['game_ids']);
```

Comment:
Adding a similar game makes it appear in the similar games list for both games, but it only appears in the Manager Similar Games page for the game it was added from.

Reply:
Oddly, the count appears to include the added game, but the list does not show it (filters are set to Any):
![image](https://github.com/user-attachments/assets/ebd7eb7d-a27e-4b13-84d6-b3715b0f921d)

Reply:
Conversely, the game where the similar game was added from shows the game in the list, but not in the count:
![image](https://github.com/user-attachments/assets/5c9670e4-43b2-472d-a40c-590d05c4d5c9)

Reply:
Fixed in latest - I needed to create a couple of new actions to ensure these bidirectional relationships are respected on attach/detach of similar games.